html/elements/input.json

```diff
@@ -1311,7 +1311,7 @@
         },
         "required": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input#required",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Attributes/required",
```

Comment:
dito

Reply:
Fixed in [036c74a](https://github.com/mdn/browser-compat-data/pull/25563/commits/036c74a1d18f9d6d352e80cc72572eb13e43513a).